### ISSUE TITLE
[AV][Audio][iOS] Sound only through headphones for iPhone 5 iOS 10.3.4

### ISSUE THREAD

**Issue by EmperorEarth** [expo/expo#7331]
## 🐛 Bug Report

`expo-av 8.0` works fine on two Android phones, Android emulator, iPhone simulator, and iPhone 6 iOS 13.3.1.

On my `iPhone 5`, `iOS 10.3.4` test device, however, the file is only audible via headphones.

Inspecting the `PlaybackStatus` object on the device, `isLoaded` is `true`, the `uri` is correct, `volume` is `1`, and when `playAsync` is called, `isPlaying` switches to `true`.

In short, everything works except speaker sound output.

Bluetooth is off. Other apps like YouTube are playing fine.

Could this be a device/iOS version compatibility issue? Is there a supported iOS/iPhone version list?

**Comment by cruzach** [comment]
Expo supports iOS 10+, so that should be supported. Could you provide a reproducible example?

**Comment by tsapeta** [comment]
Closing because of missing reproducible example.
